src/pages/Components/Flags/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Flags'
+order: 10
+---
+
+> A flag is used as a component header to categorize its content. Please use a [tag](/Components/Tags/) if you need to display additional information.
+
+A flag is composed of :
+
+- **A label :** A word or composed word.
+- **A background :** It can be solid or bordered, it allows to specify the degree of importance.
+
+## Variations
+
+<preview path="src/pages/Components/Flags/previews/flag-all"></preview>
+
+| Color Scheme   | Usage                                                 |
+| -------------- | ----------------------------------------------------- |
+| Primary        | For related content to Leroy Merlin                   |
+| Primary-Campus | For related content to Leroy Merlin Campus            |
+| Dark           | Used on light background for neutral content          |
+| Light          | Used on dark background for neutral content           |
+| Danger         | Used for sales content                                |
+
+### Two styles to make what's important stand out
+
+- Solid
+- Bordered
+
+<preview path="src/pages/Components/Flags/previews/flag-intro"></preview>
+
+## Behaviors
+
+A flag width is depending to its content.
```

Comment:
This is not a behavior. I think we can remove it

Reply:
we can remove it but we say depending *ON* by the way 😉